src/SprykerEco/Yves/Heidelpay/Hydrator/Exception/EasyCreditResponseToQuoteHydratorException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\Heidelpay\Hydrator\Exception;
+
+use Exception;
+
+class EasyCreditResponseToQuoteHydratorException extends Exception
```

Comment:
I think Exception name should display when it raises(like what kind of error raises it) instead of where.

Reply:
Converting functionality was moved in another place, so this class can be removed.